tensorflow/compiler/tf2tensorrt/convert/utils.h

```diff
@@ -73,6 +79,29 @@ struct VectorTensorShapeHasher {
    (NV_TENSORRT_MAJOR == major && NV_TENSORRT_MINOR == minor && \
     NV_TENSORRT_PATCH == patch && NV_TENSORRT_BUILD >= build))
 
+// This utility template converts an arithmetic type to a string.
+// This function is necessary to have the function:
```

Comment:
Can you re-organize these two lines to avoid partial line for the first line?

Reply:
I'm not sure I get what you mean here ?

Reply:
We limit a source line to 80 chars. A partial line is a line where we can append the first word from the next line to it without making it exceed 80 chars. In your case, you can probably move "This" from the second line to the first line.

We try to eliminate partial line in comments.

Reply:
OIC. Fixed